src/pytest_run_parallel/plugin.py

```diff
@@ -133,6 +134,14 @@ def pytest_itemcollected(item):
         n_workers = 1
         item.add_marker(pytest.mark.parallel_threads(1))
 
+    if not hasattr(item, "obj"):
+        warnings.warn(
+            f"Encountered pytest item with type '{type(item)} with no 'obj' "
```

Comment:
We can probably remove the quotes, types will probably already have angle brackets.

```suggestion
            f"Encountered pytest item with type {type(item)} with no 'obj' "
```